### ISSUE TITLE
Questions about Pi0.5 Model Training Details and High Level Planning Implementation

### ISSUE THREAD

**Issue by Ginldaj** [huggingface/lerobot#2464]
Hello, while studying the Pi0.5 model, I have two questions regarding the model implementation that I would like to ask you:
1、The paper mentions that the model adopts two-stage pre-training and designs a comprehensive loss function. However, when checking the compute_loss part in the open-source code, it is found that currently only the action loss is calculated, and the loss related to the VLM (Vision-Language Model) in the pre-training stage is not reflected. I would like to confirm whether this part is implemented elsewhere in the code or if there are other design considerations?
2、The ablation experiments in the paper show that the jointly trained Pi0.5 performs excellently in explicit and implicit High Level planning, even better than GPT4 and manual upper-level planning. However, from the open-source model code, the implementation part related to the High Level planning step has not been found for the time being. I would like to know how this part of the function is reflected in the code?
Looking forward to your reply, thank you!

**Comment by pkooij** [comment]
Hi @Ginldaj , 1. yes you are correct this part was not open sourced and thus we also didn't include it. 2. This part is also not included, Pi0.5 however still has it language head so it would be interested to actually output its intermediate text precisions and potentially (if your dataset allows) train/finetune this part as well.

**Comment by Ginldaj** [comment]
> Hi [@Ginldaj](https://github.com/Ginldaj) , 1. yes you are correct this part was not open sourced and thus we also didn't include it. 2. This part is also not included, Pi0.5 however still has it language head so it would be interested to actually output its intermediate text precisions and potentially (if your dataset allows) train/finetune this part as well.
thanks for the clear reply! Your confirmation are really helpful
